tests/test_display.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import jrnl
+import pytest
+from unittest import mock
+from jrnl.jrnl import _export_journal
+
+
+# fmt: off
+# see: https://github.com/psf/black/issues/664
+@pytest.mark.parametrize("export_format", [ "pretty", "short",])
+#fmt: on
+@mock.patch.object(argparse, "Namespace", return_value={"export": None, "filename": None})
+def test_export_format(mock_args, export_format):
+
+    test_journal = jrnl.Journal.Journal
+    mock_args.export = export_format
+    # fmt: off
+    # see: https://github.com/psf/black/issues/664
+    with mock.patch("builtins.print") as print_spy, mock.patch("jrnl.Journal.Journal.pprint") as mock_pprint:
+        _export_journal(mock_args, test_journal)
+    print_spy.call_args_list = mock_pprint
```

Comment:
Is this supposed to assert something? I'm not clear on what is tested here.

Reply:
IIRC, my intent was to test that `builtins.print()` had correctly configured call to `Journal.pprint` being passed to it. E.g. in the case of `--format pretty` I want to test that `builtins.print` is being called with `Journal.pprint(short=False)`. 

Thank you for catching this; I am fixing the test